pkg/Build/index.html

```diff
@@ -29,7 +29,7 @@
 <!-- icon font -->
 <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" rel="stylesheet">
 <!-- flavor -->
-<link rel="icon" href="../Library/assets/icons/favicon.png">
+<link rel="icon" href="../Library/Assets/icons/favicon.png">
```

Comment:
Isn't it `assets` with a lowercase "a"
https://github.com/NeonFlan/xenonjs/tree/main/pkg/Library/assets ?

Reply:
Gah, it's "Assets" in my local, which I believe is better. 

I always get in trouble with Git and case.